### PR TITLE
PT-1718, PT-1719 | feat: add is_queueing_allowed & preferred_times fields

### PR DESCRIPTION
**Pull request by karisal-anders**
### feat: add is_queueing_allowed & preferred_times fields

refs PT-1718, PT-1719

### fix: use correct Linked Events test URL in testing

replace `https://api.hel.fi/linkedevents-test/v1/` with
`https://linkedevents.api.test.hel.ninja/v1/`

refs PT-1718, PT-1719

### fix: use correct keyword set IDs (no "qq:" prefix), remove qq altogether

remove unnecessary "qq:" prefix:
- replace "qq:kultus" -> "kultus"

replace "qq" with "helsinki" in test data:
- "qq:a" -> "helsinki:a"
- "data_source": "qq" -> "data_source": "helsinki"
- "API key from data source qq" -> "API key from data source helsinki"

example kultus keyword set from production linked events:
 - https://api.hel.fi/linkedevents/v1/keyword_set/kultus:categories/
   - "id": "kultus:categories"
   - "data_source": "helsinki"

refs PT-1718, PT-1719

### docs: tidy up README.md, word wrap, fix data source, shorten wordings

refs PT-1718, PT-1719

## Related

[PT-1718](https://helsinkisolutionoffice.atlassian.net/browse/PT-1718) (is_queueing_allowed)
[PT-1719](https://helsinkisolutionoffice.atlassian.net/browse/PT-1719) (preferred_times)

Admin UI support:
 - https://github.com/City-of-Helsinki/palvelutarjotin-admin/pull/341

End-user UI support:
 - https://github.com/City-of-Helsinki/palvelutarjotin-ui/pull/312

[PT-1718]: https://helsinkisolutionoffice.atlassian.net/browse/PT-1718?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PT-1719]: https://helsinkisolutionoffice.atlassian.net/browse/PT-1719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**KULTUS-API branch is deployed to platta: https://kultus-pr348.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KULTUS-API branch is deployed to platta: https://kultus-pr348.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KULTUS-API branch is deployed to platta: https://kultus-pr348.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KULTUS-API branch is deployed to platta: https://kultus-pr348.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KULTUS-API branch is deployed to platta: https://kultus-pr348.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KULTUS-API branch is deployed to platta: https://kultus-pr348.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KULTUS-API branch is deployed to platta: https://kultus-pr348.api.dev.hel.ninja** :rocket::rocket::rocket:

**Review by nikomakela — APPROVED** [review]

File: occurrences/migrations/0043_add_is_queueing_allowed_and_preferred_times.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    dependencies = [
+        ("occurrences", "0042_auto_20231211_1254"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="palvelutarjotinevent",
+            name="is_queueing_allowed",
+            field=models.BooleanField(
+                default=True, verbose_name="is queueing to this event allowed?"
```
[the remark this answers is not in the document]
Comment (karisal-anders) [reply occurrences/migrations/0043_add_is_queueing_allowed_and_preferred_times.py]: Removed "this"

File: occurrences/models.py [hunk 2]
```diff
@@ -159,6 +159,9 @@ class PalvelutarjotinEvent(TranslatableModel, TimestampedModel):
     mandatory_additional_information = models.BooleanField(
         default=False, verbose_name=_("mandatory additional information")
     )
+    is_queueing_allowed = models.BooleanField(
+        default=True, verbose_name=_("is queueing to this event allowed?")
```
[the remark this answers is not in the document]
Comment (karisal-anders) [reply occurrences/models.py]: Removed "this"

File: occurrences/factories.py [hunk 3]
```diff
@@ -34,11 +34,16 @@ class PalvelutarjotinEventFactory(factory.django.DjangoModelFactory):
     contact_email = factory.Faker("email")
     organisation = factory.SubFactory(OrganisationFactory)
     contact_person = factory.SubFactory(PersonFactory)
+    is_queueing_allowed = True
 
     class Meta:
         model = PalvelutarjotinEvent
 
 
+class UnqueueablePalvelutarjotinEventFactory(PalvelutarjotinEventFactory):
```
[the remark this answers is not in the document]
Comment (karisal-anders) [reply occurrences/factories.py]: Removed class

**Pull request merged**